tls/extensions/s2n_client_psk.c

```diff
@@ -139,10 +139,14 @@ static S2N_RESULT s2n_client_psk_recv_identity_list(struct s2n_connection *conn,
         GUARD_AS_RESULT(s2n_blob_init(&identity, identity_data, identity_size));
 
         /* TODO: Validate obfuscated_ticket_age when using session tickets:
-         *       https://github.com/awslabs/s2n/issues/2417
          *
-         * "For identities established externally, an obfuscated_ticket_age of 0 SHOULD be
-         * used, and servers MUST ignore the value."
+         *       https://github.com/awslabs/s2n/issues/2417
+         */
```

Comment:
You can probably just drop the TODO. It's basically, "TODO: session resumption".